packages/editor/src/components/list-view-sidebar/style.scss

```diff
@@ -22,11 +22,17 @@
 	// This allows items in the list view to have equidistant padding left and right
 	// right up until a scrollbar is present.
 	scrollbar-gutter: auto;
+}
 
-	// The table cells use an extra pixels of space left and right. We compensate for that here.
+.editor-list-view-sidebar__list-view-panel-content {
 	padding: $grid-unit-05;
 }
 
+// Match the Outline panel’s padding with the surrounding panels.
+.editor-list-view-sidebar__list-view-container > .document-outline {
+	padding: $grid-unit-20;
+}
```

Comment:
Let's nest this inside of `.editor-list-view-sidebar__list-view-container` so that's better organized, we already have that in place.

The change should look something like this:

```
.editor-list-view-sidebar__list-view-container {
	// rules that are already there
 	> .document-outline {
		// rules you added here
	}
}
```